src/main/java/com/hackclub/hccore/slack/SlackBot.java

```diff
@@ -493,4 +548,14 @@ public boolean sendVerificationMessage(String id, String mcName, String uuid) th
       return false;
     }
   }
+
+  public String generateVerificationCode(String mcUuid) {
+    if (mcUuidVerificationCodes.containsKey(mcUuid)) {
```

Comment:
i have the opinion that it should regenerate the code if requested

Reply:
Like it gives you a new code every time you try to rejoin?

Reply:
Maybe not with expiry implemented
You should be able to instead refresh the code's time, by getting the current value and re-setting it